src/webgl/p5.RendererGL.js

```diff
@@ -631,7 +635,22 @@ p5.RendererGL.prototype._getColorShader = function () {
   return this._defaultColorShader;
 };
 
+p5.RendererGL.prototype.getTexture = function (img) {
+  // efficiency?
+  var f = function(element) {
+    return element.src === img;
+  };
+
+  var tex = this.textures.find(f);
+  if (tex === undefined) {
+    tex = new p5.Texture(this, img);
+    this.textures.push(tex);
+  }
 
+  tex.set(img);
```

Comment:
Do we need to set the image if we are just fetching it?

I think that updating the texture with new pixels should probably be a separate concern from just getting the texture for rendering.